### PR TITLE
Add requirements docs and fix link to the procedure to add new packages

### PR DESCRIPTION
**Pull request by amoralej**
It has been moved to a separate doc.

### CODE REVIEW

File: source/documentation/requirements.html.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+1. If the project follows global-requirements processes, make sure that the requirement has been
+added to global-requirements.txt and upper-constraints.txt files as described in the [upstream
+documentation](https://github.com/openstack/requirements/#proposing-changes)
+    
+    <br />
+2. Check if the new requirement is present in CentOS base channels. The easies way to do this
+is using yum command from a system running CentOS 7:
+    
+        yum list "*<dependency>"
```
Comment (apevec) [inline source/documentation/requirements.html.md]: Too bad that provides pythonVERdist(PYPI_NAME) are only available in Fedora >= 25
https://fedoraproject.org/wiki/Packaging:Python#Automatic_Provides_with_a_standardized_name

File: source/documentation/requirements.html.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+search expression. If you find the desired dependency, you can open a bug in [Red Hat Bugzilla for
+RDO product](https://bugzilla.redhat.com/enter_bug.cgi?product=RDO&component=distribution) requesting
+the inclussion of the package in RDO repos. RDO Core members will handle the request.
+    
+    <br />
+5. If the new package is not in CBS, you must check if it's packaged in Fedora using the [package
+browser](https://apps.fedoraproject.org/packages/). If the package exists you can open a bug in [Red Hat Bugzilla for
+RDO product](https://bugzilla.redhat.com/enter_bug.cgi?product=RDO&component=distribution) requesting
+the rebuild of the package in RDO repos. RDO Core members will rebuild the package using latest existing version
+in fedora.
```
Comment (apevec) [inline source/documentation/requirements.html.md]: Fedora

File: source/documentation/requirements.html.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+    
+    <br />
+6. When the packages doesn't exist even in fedora you need to add the package following the [New package
```
Comment (apevec) [inline source/documentation/requirements.html.md]: Fedora

File: source/documentation/requirements.html.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+process](https://fedoraproject.org/wiki/New_package_process_for_existing_contributors). Note that a fedora
```
Comment (apevec) [inline source/documentation/requirements.html.md]: Fedora

File: source/documentation/requirements.html.md [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+7. Once the package is available in the repos, you can add it to the list of *Requires* or *BuildRequires* in
+your package spec file. Note that optional dependencies not used in default or common configurations should
+not be added as Requires but installed only when needed.
+
+
+### Updating a requirement in RDO CloudSIG repositories
+
+There are some rules to follow when a requirement update is needed by a OpenStack project:
+
+* If the dependency is included in upstream requirements project, the required version must be lower or equal to
```
Comment (apevec) [inline source/documentation/requirements.html.md]: ideally, it should be equal u-c

If mentioning lower, g-r >= must be mentioned along with caveat that g-r lower bounds are not tested upstream and might not be working!

File: source/documentation/requirements.html.md (hunk 1 above)
Comment (amoralej) [inline source/documentation/requirements.html.md]: that'd be nice but I couldn't figure out something better for centos. Additionally, there is no web based package database (or at least i couldn't find it) that i can link.

File: source/documentation/requirements.html.md [hunk 6]
```diff
@@ -38,8 +38,8 @@ If you have questions or special requests, don't hesitate in contacting RDO usin
 
 ### Adding a new requirement to RDO
 
-When a new requirement is needed for a OpenStack project included in RDO, following process
-must be followed by package maintainers:
+When a new requirement is needed for an OpenStack project included in RDO, package maintainers
+must follow by package maintainers:
```
Comment (apevec) [inline source/documentation/requirements.html.md]: s/by package maintainers/this procedure/

Comment (amoralej) [inline source/documentation/requirements.html.md]: Fixed

**Pull request merged**